User_Story#1/SortingApp/SortinfApp.UnitTests/Utilities/DataReaderTest.cs

```diff
@@ -0,0 +1,118 @@
+﻿using System;
+using System.IO;
+using BusinessLayer.Reader;
+using BusinessLayer.Utilities.Validator.Interfaces.Fakes;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace SortinfApp.UnitTests.Utilities
+{
+    /// <summary>
+    /// Represents tests for DataReader class.
+    /// </summary>
+    /// <owner>Anton Petrenko</owner>
+    [TestClass]
+    public class DataReaderTest
+    {
+        /// <summary>
+        /// Tests DataReader if pass empty validator in constructor it will throw ArgumentNullException.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        [TestMethod]
+        public void DataReader_DataReader_PassEmptyValidatorInConstructor_ThrowArgumentNullException()
+        {
+            //
+            // Assert.
+            //
+            Assert.ThrowsException<ArgumentNullException>(() => new DataReader(null));
+        }
+
+        /// <summary>
+        /// Tests ReadContent if pass empty path to the file it will throw ArgumentNullException.
+        /// </summary>
+        /// <owner>Anton Petrenko</owner>
+        [TestMethod]
+        public void DataReader_ReadContent_PassEmptyPathToTheFile_ThrowArgumentNullException()
+        {
+            //
+            // Arrange.
+            //
+            var validator = new StubIValidator()
+            {
+                IsDataExistString = (string str) => throw new ArgumentNullException(nameof(str))
```

Comment:
In such case you have two asserts in a single test method.
Asserts should be separated.

Reply:
Please write me if you have questions.

Reply:
I've slightly changed test case in Arrange part, but I don't understand how to avoid using both of these initialization.